log/log.go

```diff
@@ -5,12 +5,27 @@ import (
 	"github.com/ethereum/go-ethereum/metrics"
 )
 
+// Action Defines swap log actions
+type Action string
```

Comment:
The solution I found was using a type alias for passing constant like variables.

The idea was to use always the same type of action, but since there are too many variations, I resolved to remove actions altogether.